cmds/cpud/serve.go

```diff
@@ -115,6 +116,15 @@ func serve() error {
 	if err != nil {
 		return err
 	}
+
+	if *mdnsEnabled {
+		mdns_server, err := server.Advertise(*port, *mdnsTxt)
+		if err != nil {
+			return fmt.Errorf("Could not advertise with mdns: %w", err)
+		} else {
```

Comment:
I don't think this `else` is needed because the previous `if` statement `return`s
```
if err != nil {
  return fmt.Errorf(....)
}
defer mdns_server.Shutdown()
```

Reply:
true, will simplify.